src/VirtoCommerce.ExperienceApiModule.Core/VirtoCommerce.ExperienceApiModule.Core.csproj

```diff
@@ -31,7 +31,7 @@
         <PackageReference Include="VirtoCommerce.CoreModule.Core" Version="3.2.0" />
         <PackageReference Include="VirtoCommerce.TaxModule.Core" Version="3.2.0" />
         <PackageReference Include="VirtoCommerce.SearchModule.Core" Version="3.13.0" />
-        <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.57.0" />
+        <PackageReference Include="VirtoCommerce.Platform.Core" Version="3.66.0" />
```

Comment:
Update `<PackageReference Include="VirtoCommerce.Platform.Security" Version="3.57.0" />` to `<PackageReference Include="VirtoCommerce.Platform.Security" Version="3.66.0" />`.